src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -399,7 +409,7 @@ function PaymentMethodList({
                             icon={Expensicons.CreditCard}
                             onPress={onPress}
                             // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-                            isDisabled={isLoadingPaymentMethods || isFormOffline}
+                            isDisabled={isLoadingPaymentMethods || isFormOffline || isLoadingPaymentMethodsOnyx}
```

Comment:
@dominictb Why do we need isLoadingPaymentMethodsOnyx? isLoadingPaymentMethods should do the work here, right or am i missing something?

Reply:
Thanks for the details @dominictb.

I noticed an issue on the Expensify cards page within the workspaces—pressing the close icon doesn’t clear the error message. Although it's not related to the current issue but we had fixed a similar problem here in this PR.so, Just to confirm, @dominictb @cead22  should we address it as part of this task, or create a separate bug for it?

https://github.com/user-attachments/assets/b0f078a4-e731-4bb0-99d6-897bea23a118

Reply:
Hi! Coming in here as assigned - if it's going to be a pretty similar solution let's knock it out in this PR as well. I realize it's a different page but still card-related error I guess, and if we already have done the work we need to adjust, let's do it.